packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -293,6 +295,7 @@ class IOSDevice extends Device {
   final Platform _platform;
   final IMobileDevice _iMobileDevice;
   final IProxy _iproxy;
+  final bool _devModeEnabled;
```

Comment:
Looks like there's no need for this private variable. It does not appear to be used anywhere. I would remove it and in the constructor args use `required this.devModeEnabled`